From: Qingqing <wangyiqing@kylinos.cn>
Date: Fri, 17 Jul 2026 07:52:34 +0000
Subject: Translated using Weblate (Korean)

Currently translated at 100.0% (3 of 3 strings)

Translation: openkylin-huanghe/peony-set-wallpaper
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-set-wallpaper/ko/
---
 .../translations/peony-set-wallpaper-extension_ko.ts                | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/peony-set-wallpaper/translations/peony-set-wallpaper-extension_ko.ts b/peony-set-wallpaper/translations/peony-set-wallpaper-extension_ko.ts
index bb93418..b214354 100644
--- a/peony-set-wallpaper/translations/peony-set-wallpaper-extension_ko.ts
+++ b/peony-set-wallpaper/translations/peony-set-wallpaper-extension_ko.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../set-wallpaper-plugin.h" line="42"/>
         <source>Peony-Qt set wallpaper Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>파일관리자 배경화면 플러그인 설정</translation>
     </message>
     <message>
         <location filename="../set-wallpaper-plugin.h" line="43"/>
         <source>Set wallpaper Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>배경화면 플러그인 설정</translation>
     </message>
     <message>
         <source>set wallpaper Extension</source>
@@ -24,7 +24,7 @@
     <message>
         <location filename="../set-wallpaper-plugin.cpp" line="65"/>
         <source>Set as wallpaper</source>
-        <translation type="unfinished"></translation>
+        <translation>배경화면으로 설정</translation>
     </message>
 </context>
 </TS>
